app/Http/Controllers/UserController.php

```diff
@@ -2,10 +2,94 @@
 
 namespace App\Http\Controllers;
 
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Validator;
+
 class UserController extends Controller
 {
     public function index()
     {
-        return view('auth.user');
+        $user = Auth::user();
+
+        return view('auth.user', ['user' => $user])
+            ->with('neptun', $user->educationalInformation->neptun)
+            ->with('phone_number', $user->personalInformation->phone_number)
+            ->with('faculties', $user->faculties)
+            ->with('workshops', $user->workshops);
+    }
+
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    public function updateEmail(Request $request)
+    {
+        $user = Auth::user();
+
+        $validator = Validator::make($request->all(), [
+            'email' => 'required|email|max:225|unique:users',
+        ]);
+        $validator->validate();
+
+        if ($validator->fails()) {
+            return redirect()->back()
+                ->withErrors($validator)
+                ->withInput();
+        }
+        $user->update([
+            'email' => $request->email,
+        ]);
+
+        return redirect()->back()->with('message', 'ok');
```

Comment:
Is the message used?

Reply:
Not yet, I want to standardize this in a new issue. For now, I will delete.

Reply:
Okay, great! You might want to look if there is some existing solution?